pg_store_plans.c

```diff
@@ -1216,6 +1216,8 @@ pgsp_store(char *plan, queryid_t queryId,
 
 	Assert(plan != NULL && queryId != PGSP_NO_QUERYID);
 
+	memset(&key, 0, sizeof(pgspHashKey));
```

Comment:
This should be ifdef'd for pg11+ as until pg10 it was still a uint32, since the rest of the code correctly handle the version change.